planetiler-custommap/samples/owg_simple.yml

```diff
@@ -0,0 +1,160 @@
+schemaName: OWG Simple Schema
```

Comment:
I was thinking that we didn't want the samples packaged in the jar, though maybe for important ones like openmaptiles (someday) and the owg schema (if it matures) it might make sense for them to be pre-configured options?